### PR TITLE
Propose branch and release strategies

### PR DESCRIPTION
**Pull request by gsong**
[View rendered version](https://github.com/EnterpriseDB/docs/tree/proposal/george/branch-and-release/docs/proposals/branch-and-release).

### CODE REVIEW

File: docs/proposals/branch-and-release/README.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+PR][pr state].
+
+## Use Cases
+
+### Hot Fix
+
+Stop the presses, something is seriously wrong with
+<https://www.enterprisedb.com/docs/>.
+
+-  **Base branch**: `main`
```
Comment (epbarger) [inline docs/proposals/branch-and-release/README.md:116]: Is this for the situation where there is somehow an issue that only affects `main`, but not `develop`? If that's not the case, with this strategy `develop` would be missing the hot fix, correct?

Comment (gsong) [reply docs/proposals/branch-and-release/README.md:116]: This is meant to address the case of a production emergency case, when we're not ready to release `develop` yet.

<https://github.com/EnterpriseDB/docs/pull/915/files#diff-47f702b951edf20e1664629415407877c1f90a3e198c871989021b8bd707e2b5R59> ensures that hot fix changes make it back to `develop`.

**Review by epbarger — APPROVED** [review]
This all looks good to me

**Comment by gsong** [comment]
@josh-heyer @cero-miedo Thoughts around this? Brad, Shanon, and Evan all think it looks good.

**Review by josh-heyer — APPROVED** [review]
Looks solid to me. Printing it out & taping it to my wall; let's put 'er into action & file off any rough edges!

**Pull request merged**